### DISCUSSION TITLE
Provide UI indication that RagnaDemoAssistant option is not real

### DISCUSSION THREAD

**Discussion by droumis** [Quansight/ragna#142] [General]
First, great work! This looks amazing.

Many people reading the [Medium blog post](https://medium.com/quansight/unveiling-ragna-an-open-source-rag-based-ai-orchestration-framework-designed-to-scale-from-45a6ee07d2a8) will follow those simple instructions to get started and choose Ragna/DemoAssistant - the only option available to them out of the box. They might be disappointed to find that the answers are not useful. 

I'd suggest adding a warning of some sort for this option in the UI, as is present in the actual documentation: "RagnaDemoAssistant - This is not actually an LLM, and will not provide useful answers. It's a demo (toy) assistant so that you can quickly setup and use Ragna."

Maybe even a pointer from there about how to add real LLMs.

**Comment by pmeier** [comment]
Hey @droumis and thanks for the suggestion. We made some changes since the initial release that hopefully makes it a little more transparent. Have a look at the issue #124 and the our solution in #125. TL;DR `ragna ui` will no longer work without creating a configuration file first. And by "forcing" users to go through the config wizard, the "demo means demo" should be explicit.
